core/dbt/lib.py

```diff
@@ -89,14 +88,18 @@ def get_dbt_config(project_dir, args=None, single_threaded=False):
         target=getattr(args, "target", None),
     )
 
+    # set global flags from arguments
```

Comment:
The functions in this file are still by large untested, gonna revisit to replace this file and test the interfaces we have to replace it.